modules/secure-ci/variables.tf

```diff
@@ -119,3 +109,9 @@ variable "cloudbuild_private_pool" {
   type        = string
   default     = ""
 }
+
+variable "clouddeploy_pipeline_name" {
+  description = "Cloud Deploy pipeline name"
+  type        = string
+  default     = ""
```

Comment:
Should this null by default? Seems like that would make the `_CLOUDDEPLOY_PIPELINE_NAME` subst empty.